README.md

```diff
@@ -18,3 +18,27 @@ However, expect things to break or change without warning.
 
 See here for an example: https://github.com/pymc-devs/pymc4/blob/master/notebooks/radon_hierarchical.ipynb
 See here for the design document: https://github.com/pymc-devs/pymc4/blob/master/notebooks/pymc4_design_guide.ipynb
+
+## Develop
+
+One easy way of developing on PyMC4 is to take advantage of the development containers! 
+Using pre-built development environments allows you to develop on PyMC4 without needing to set up locally.
+
+To use the dev containers, you will need to have Docker and VSCode running locally on your machine, and will need the VSCode Remote extension installed inside VSCode.
+
+Once you have done that, to develop on PyMC4, on GitHub:
+
+1. Make a fork of the repository
+2. Create a new branch inside your fork
+3. Copy the branch URL
+
+Now, in VSCode:
+
+1. In the command palette, search for "Remote-Containers: Open Repository in Container...".
+2. Paste in the branch URL
+3. If prompted, create it in a "Unique Volume".
+
+Happy hacking away! Don't forget to commit your changes and push them to your branch.
```

Comment:
If the changes are really lost when VSCode is closed, the "Don't forget to.." warning should be more prominent

Reply:
Done!